ProcessSlave.php

```diff
@@ -258,7 +258,7 @@ public function run()
     {
         $this->loop = \React\EventLoop\Factory::create();
 
-        $this->errorLogger = new BufferingLogger();
+        $this->errorLogger = BufferingLogger::create();
         ErrorHandler::register(new ErrorHandler($this->errorLogger));
```

Comment:
Doesn't ErrorHandle wants a `\Symfony\Component\Debug\BufferingLogger` instance which fails when you pass a `PHPPM\Debug\ BufferingLogger`?